python/cudf/cudf/tests/test_array_function.py

```diff
@@ -67,6 +67,8 @@ def test_array_func_cudf_series(np_ar, func):
         lambda x: np.sum(x, axis=0),
```

Comment:
Should we explicitly test `product` (with various `axis` options?), since its dispatches to CuPy are special? Does that need to be tested with NumPy dispatches as well as "normal" pandas calls?

Reply:
Yup, added tests for this.